src/ja/worker/docker_events.py

```diff
@@ -0,0 +1,23 @@
+"""
+This module is responsible for the monitoring of the docker containers
+running on the worker machine.
+"""
+
+
+class DockerEvents:
+    """
+    Monitors the containers on the current machine
+    """
+
+    def __init__(self) -> None:
+        """!
+        Starts the event loop
+        that will be monitoring the state of the containers.
+        """
+
+    def change_filters(self, filters: dict) -> None:
+        """!
+        Change on which events the server is going to be notified
+        :param filters: filter the events by event time, container or image
```

Comment:
we use `@` in front in the other places (see my and @JohannesGaessler 's pull requests)